RileyLink.xcodeproj/project.pbxproj

```diff
@@ -2926,7 +2929,7 @@
 					};
 					C1FFAF7F213323CC00C50C1D = {
 						CreatedOnToolsVersion = 9.4.1;
-						DevelopmentTeam = UY678SP37Q;
+						DevelopmentTeam = 86HTWX383F;
```

Comment:
Seems like there are still a few of these?

Reply:
I removed them from the UI but apparently there was still one left. Thanks for catching that.

Reply:
There are a couple more at the bottom of the diff.